app/scripts/modules/core/src/pipeline/config/stages/stage.module.js

```diff
@@ -197,6 +202,20 @@ module.exports = angular.module('spinnaker.core.pipeline.config.stage', [
       }
     }
 
+    function stageExcludesAllAccounts(stageType, accounts) {
+      if (stageType.excludedCloudProviders) {
+        const anyNotExcluded = some(accounts, account => {
```

Comment:
Would use native `[].some` here and below.

Reply:
Would also be a little more readable if the bool were swapped such that this was an `[].every`. i.e., every `account` is excluded by at least one of the `excludedCloudProviders`.

Reply:
done